.gitignore

```diff
@@ -23,3 +23,4 @@ bin/
 *.graphml
 coverage.db*
 clover.license
+local.properties
```

Comment:
@marcphilipp - good catch - the file was created in error (by Android Studio - don't ask). I've removed this line.